04 Abstraction Exercises Part 1/music-player-abstraction-exercise/src/com/music/player/abstraction/exercise/amulong/MusicPlayer.java

```diff
@@ -0,0 +1,46 @@
+package com.music.player.abstraction.exercise.amulong;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MusicPlayer {
+    private static List<String> playlist;
```

Comment:
Why do you have 2 playlists?

Reply:
Sir sabi nyo po kahapon mag 2 playlist po with same content po

Reply:
dalawang playlist or dalawang music player ang sinabi ko?